src/components/card/card.stories.mdx

```diff
@@ -536,7 +535,7 @@ This example uses the `react-dnd` and `react-dnd-html5-backend` packages.
 
```

Comment:
You can probably remove this now as you have resolved the issue with it only working in `Canvas`
https://github.com/Sage/carbon/pull/4735/files#diff-2f88d95fbb4280b0323e6304d1d46dd6b6900d1c98a38e104167323b5452351bR530